recipe/meta.yaml

```
@@ -13,11 +13,15 @@ source:
 
 build:
   number: 0
-  script: pip install . --no-deps --ignore-installed --no-cache-dir
+  script: ${PYTHON} -m pip install . --no-deps --ignore-installed --no-cache-dir
```

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --vv
```
Looks like this needs the Jinja-templated `PYTHON` instead of the Unix-style one you used, and the default arguments for pip have been simplified too.